src/commands/server/update.ts

```diff
@@ -35,11 +33,13 @@ export default class Update extends Command {
       char: 'a',
       description: 'Installer type. If not set, default is autodetected depending on previous installation.',
       options: ['operator', 'olm'],
+      hidden: true,
     }),
     platform: string({
       char: 'p',
       description: 'Type of Kubernetes platform. Valid values are \"minikube\", \"minishift\", \"k8s (for kubernetes)\", \"openshift\", \"crc (for CodeReady Containers)\", \"microk8s\".',
       options: ['minikube', 'minishift', 'k8s', 'openshift', 'microk8s', 'docker-desktop', 'crc'],
+      hidden: true,
```

Comment:
Isn't it a required parameter?

Reply:
No, it is not required. 
We hide it since it isn't used.

Reply:
Maybe, then, we have to delete it at all?

Reply:
Let's keep it for backward compatibility